ga.py

```diff
@@ -33,8 +40,28 @@ def assigned_priorities(prios, phenotype):
     return np.array(assigned_prios)
 
 
-def compute_costs(prios, phenotype):
-    return np.sum(assigned_priorities(prios, phenotype))
+def punish_erroneously_assigned_talkers(talkers, phenotype):
+    cost = [] # cost for talks where talker is not present in talks
+    for talker, talk in talkers.items():
+        c = 0
+        #punish if their first prio is not in the same slot as their talk
+        if talk in phenotype[0][0]:
+            if 0 in phenotype[1][0]:
+                c += 1000
+            #talkers should get a prio that is not in the same slot as their talk --> punish if this isnt the case
+            if 1 in phenotype[0][0] or 2 in phenotype[0][0]: 
+                c += 10
+        else:
+            if 0 in phenotype[0][0]:
+                c += 1000
+            if 1 in phenotype[1][0] or 2 in phenotype[0][0]:
+                c += 10
+        cost.append(c)
+    return np.array(cost)
+
+
+def compute_costs(prios, talkers, phenotype):
+    return np.sum(assigned_priorities(prios, phenotype)) + np.sum(punish_erroneously_assigned_talkers(talkers, phenotype))
```

Comment:
Please break up long lines (in this case, probably use parentheses around the `+` sum and then break the line after `+`.